server/eventlog_test.go

```diff
@@ -259,7 +269,10 @@ func TestParseEventLogs(t *testing.T) {
 			subtestName := fmt.Sprintf("%s-%s", log.name, hashName)
 			t.Run(subtestName, func(t *testing.T) {
 				if _, err := ParseMachineState(rawLog, bank); err != nil {
-					t.Errorf("failed to parse and replay log: %v", err)
+					gErr := err.(*GroupedError)
```

Comment:
It *should* be a `GroupedError`. `ParseMachineState` should always return an error and that should be part of our interface. That said, I'm happy to check the output of the test here and make it explicitly part of our interface rather than have the test panic.